packages/@uppy/dashboard/src/components/FileItem.js

```diff
@@ -128,40 +128,51 @@ module.exports = function FileItem (props) {
           onPauseResumeCancelRetry={onPauseResumeCancelRetry}
           file={file}
           error={error}
+          isUploaded={isUploaded}
           {...props} />
```

Comment:
Great, spent about an hour trying to figure out why the progress indicator remained a `<button>` on master, but turned into a `<div>` in your PR, if the PR is not touching the logic of that component 🙀 😂